Gemfile

```diff
@@ -13,6 +13,7 @@ gem 'fabrication' # Fabrication generates objects in Ruby. Fabricators are schem
 gem 'sidekiq' # background processing for Ruby
 gem 'bootsnap', require: false # Reduces boot times through caching; required in config/boot.rb
 gem 'i18n-js', '3.5.1' # A library to provide the I18n translations on the Javascript
+gem 'httparty' # A library to call external API
```

Comment:
We are not using it in this PR, so it can be added in this https://github.com/suho/google-search-ruby/pull/45 PR

Reply:
Fixed in [`4c447f6` (#42)](https://github.com/suho/google-search-ruby/pull/42/commits/4c447f699243eec5c046c00dd38c76964fc91c8f)